anchor/common/ssv_types/src/cluster.rs

```diff
@@ -0,0 +1,53 @@
+use crate::OperatorId;
+use crate::Share;
+use derive_more::{Deref, From};
+use types::{Address, Graffiti, PublicKey};
+
+/// Unique identifier for a cluster
+#[derive(Clone, Copy, Debug, Default, Eq, PartialEq, Hash, From, Deref)]
+pub struct ClusterId(pub u64);
```

Comment:
I know there is some terminology overlap but it's probably better we stick with "committee" for now so we're using the same terminology as the other client team

Reply:
An argument for "cluster" is that "committee" might be confusable with the committees from Ethereum consensus

Reply:
Yep Daniel mentioned my idea behind Cluster. Im good to change it but in that case we should name it something like SSVCommittee just to be explicit about the difference.